import_users/import_users.rb

```diff
@@ -61,7 +61,8 @@ def add_user(name, email, role, title)
       :user => {
         :name => name,
         :email => email,
-        :role => role,
+        # .downcase and strip to handle inconsistencies in the .csv
```

Comment:
You can remove the comment from the code and push your change again. You can look at this [Ruby style  guide](https://github.com/bbatsov/ruby-style-guide#comments) for reference.